ansible_collections/arista/avd/plugins/plugin_utils/errors/errors.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, message: str = "Schema Error", error=None):
             raise AristaAvdError('Python library "jsonschema" must be installed to use this plugin') from JSONSCHEMA_IMPORT_ERROR
 
         if isinstance(error, (jsonschema.ValidationError)):
+            self.path = self._json_path_to_string(error.absolute_path)
```

Comment:
I think we should just take the `error.absolute_path` instead of having it as a string.

Reply:
it will be a list then, eg. deque(['router_bgp', 'peer_groups', 0, 'link_bandwidth'])